app/cafe.py

```diff
@@ -0,0 +1,23 @@
+import datetime
+from app.errors import (NotVaccinatedError, OutdatedVaccineError,
```

Comment:
Code Style: It is recommended to use `( )` while importing multiple classes/modules. This will make your imports more readable and maintainable. Here is how you can do it:
python
from app.errors import (
    NotVaccinatedError,
    OutdatedVaccineError,
    NotWearingMaskError
)
```